### DISCUSSION TITLE
One-to-many relation on children table

### DISCUSSION THREAD

**Discussion by thangngoc89** [pocketbase/pocketbase#4831] [Q&A] [answered]
For example Post -> Comments.
I usually have post_id on comments table and do a join when I need to get Post and comments at the same time.

However, from reading Pocketbase docs, it seems like I need to have a multi relations field on Post for the comments. This means that I have to update the field on Post which could lead to race condition when there are several comments coming at the same time.

Is there anyway to achieve this instead?

**Comment by ganigeorgiev** [comment, marked as answer]
> It seems like I need to have a multi relations field on Post for the comments.

No. If you are asking for expanding relations, `expand` works both ways - see https://pocketbase.io/docs/working-with-relations/#back-relations.

If that's not what you are trying to do, please provide more details about your collections structure and what is your use case because from just the above description is not clear.

**Reply by thangngoc89** [reply]
Thanks. Exactly what I needed
